lib/noether/either.ex

```diff
@@ -193,6 +193,33 @@ defmodule Noether.Either do
   def map_error(a = {:ok, _}, _), do: a
   def map_error({:error, a}, f) when is_function(f, 1), do: {:error, f.(a)}
 
+  @doc """
+  Given a list of values and a function returning `{:ok, any}` or `{:error, any}`, it applies the function on every
+  `value` returning `{:ok, values}` if every `f.(v)` results in `{:ok, v}`; returning `{:error, _}` if `f.(v) results in
+  an `{:error, any}.
+
+  ## EXAMPLES
+    iex> map_all(["23:50:07.0123456", "23:50:07.123Z"], &Time.from_iso8601/1)
```

Comment:
Sorry to bother you on such small details... please note the formatting of other comments. It should be:

```
## Examples

      iex> ...
```

- Examples is not UPPERCASE
- If memory serves me right, `iex>` is indented with 6 spaces. This is critical for the formatting of `hex.docs`.